www/blog/2022-11-06-announcing-trpc-10.mdx

```diff
@@ -0,0 +1,82 @@
+---
+slug: announcing-trpc-10
+title: Announcing tRPC 10
+author: Alex / KATT 🐱
+author_title: Creator of tRPC
+author_url: https://twitter.com/alexdotjs
+author_image_url: https://avatars1.githubusercontent.com/u/459267?s=460&v=4
+---
+
+## Introduction
+
+tRPC's mission from day one has been to provide you with an easy way of moving fast and not breaking nothing whilst building full-stack applications.
```

Comment:
I dislike this

Reply:
tRPC's purpose is to provide you with an easy way to rapidly(quickly?) build full stack applications without breaking anything.

maybe that will spark alternatives in your mind

Reply:
Here's my take if it's useful:

tRPC aims to provide the best full stack application by enforcing tight type bindings between backend and frontend - without the codegen, drift between API contracts or time wasted unit testing things that are instead bound with TypeScript contracts.